test/CMakeLists.txt

```diff
@@ -35,13 +35,13 @@ macro (add_app_test test_filename test_alternative)
     # Create the test target.
     add_executable (${target} ${test_filename})
     target_link_libraries (${target} "${PROJECT_NAME}_lib" seqan3::seqan3 gtest_all pthread)
+    target_include_directories ("${target}" PUBLIC ${CMAKE_SOURCE_DIR}/lib/seqan3/test/include)
```

Comment:
I would put this change into a separate `[INFRA]` commit. This way you can later on find all you changes to the infra structure more easily, instead of them being mixed with other normal commits.